modules/ROOT/pages/reference/procedures.adoc

```diff
@@ -992,18 +982,8 @@ m|READ
 |===
 | Description
 a|
-Query the given full-text index.
-
-Returns the matching relationships and their Lucene query score, ordered by score.
-
-Valid _key: value_ pairs for the `options` map are:
-
-* `skip: <number>` -- skip the top N results.
-* `limit: <number>` -- limit the number of results returned.
-* `analyzer: <string>` -- use the specified analyzer as a search analyzer for this query.
-
-The `options` map and any of the keys are optional.
-An example of the `options` map: `{skip: 30, limit: 10, analyzer: 'whitespace'}`
+Query the given full-text index. Returns the matching relationships, and their Lucene query score, ordered by score.
+Valid keys for the options map are: 'skip' to skip the top N results; 'limit' to limit the number of results returned; 'analyzer' to use the specified analyzer as search analyzer for this query.
```

Comment:
```suggestion
Valid _key: value_ pairs for the `options` map are: `skip: <number>` - to skip the top N results; `limit: <number>` - to limit the number of results returned; `analyzer: <string>` - to use the specified analyzer as a search analyzer for this query.
The `options` map and any of the keys are optional.
An example of the `options` map: `{skip: 30, limit: 10, analyzer: 'whitespace'}`
```

Reply:
I do like this change.

I do not mind the use of dashes, though the hyphens should probably be em-dashes (`--`, in ascii-doc; `—`, in unicode)

However, they are using within a list format which uses "super comma" semicolons  (for lack of a better word).
These dashes probably should be _commas_ to keep the standard way of listing with sublists or with clarification within prose. We also shouldn't forget the final _and_.

```suggestion
Valid _key: value_ pairs for the `options` map are: `skip: <number>`, to skip the top N results; `limit: <number>`, to limit the number of results returned; and `analyzer: <string>`, to use the specified analyzer as a search analyzer for this query.
The `options` map and any of the keys are optional.
An example of the `options` map: `{skip: 30, limit: 10, analyzer: 'whitespace'}`
```

---

Though I am aware of the docs team's preference to avoid semicolons.
Though not my preference, perhaps we should rework this to not require the semicolons.

```suggestion
Valid _key: value_ pairs for the `options` map are: `skip: <number>` (to skip the top N results), `limit: <number>` (to limit the number of results returned), and `analyzer: <string>` (to use the specified analyzer as a search analyzer for this query).
The `options` map and any of the keys are optional.
An example of the `options` map: `{skip: 30, limit: 10, analyzer: 'whitespace'}`
```
Though I personally do not think using parentheses helps here, perhaps there is a better way, by reverting back to a more explicit list form like it was originally.

Reply:
TBH, I'd prefer to continue with hyphens/em-dashes (I like em-dashes even more). I think, commas will make it even more difficult to read this list, parentheses - too complicated and I don't see any reason for them.
Vertical lists are better for readability, so I'm suggesting to return to the original way of displaying those lists.

Reply:
I agree that parantheses just make things silly.
Though I do like the comma-semicolon lists, I recognise that many are unfamiliar with them and the vertical listing is better.

We should be able to accommodate the in the doc strings.
However, I am unsure how you test the equivalence of the docstrings to that of the docs. If it is a simple plain text comparison then we can quite simply write the docstring with the exact same asciidoc

Reply:
Sorry for a long silence on this PR. So, as our test expert said using the same asciidoc is the best way. The testing parses asciidoc. I suggest to continue with vertical listing.